lib/AST/ASTImporter.cpp

```diff
@@ -8452,6 +8463,7 @@ Decl *ASTImporter::MapImported(Decl *From, Decl *To) {
   // This mapping should be maintained only in this function. Therefore do not
   // check for additional consistency.
   ImportedFromDecls[To] = From;
```

Comment:
Maybe add a FIXME that the function can be called multiple times with same `To` and different `From`.